source/vocab/details.ttl

```diff
@@ -919,6 +920,13 @@
     rdfs:domain :Note;
     owl:equivalentProperty bf2:noteType .
 
+:physicalDetailsNote a owl:DatatypeProperty ;
+    rdfs:label "Physical details note"@en, "Övriga fysiska detaljer"@sv ;
+    rdfs:domain :Instance ;
+    rdfs:Range rdfs:Literal ;
```

Comment:
Do we usually add `rdfs:range` to `owl:DatatypeProperty`? But if we do, it should be with lowercase: `rdfs:range` 😄